cypress/support/page-objects/base-page.js

```diff
@@ -7,11 +7,14 @@ export default class BasePage {
         }
```

Comment:
It doesn't appear these selectors are being used and also overwritten in files extending the class - can you talk me through this

Reply:
You make a valid point. Since it's not currently using any common selectors: Remove the empty selectors object and add it only when needed